fixes to make it work with latest wsdl2phpgenerator version

ivoba opened the pull request:
**PR Update 3.2.2017**
- raised PHP to 5.5.3 since its the min PHP version of symfony 2.3
- updated symfony console to 2.6 since its the highest version that supports PHP 5.3.3
- added PHP7 & PHP7.1 to travis
- some cleanups regarding wsdl2phpgenerator v3
- updated dependencies where possible
- updated wsdl2phpgenerator to 3.4

- ~~updated wsdl2phpgenerator to 3.1.2
  in the composer.lock there was a older version that worked in the project. If you would use it inside another project however it would fail because the newest wsdl2phpgenerator was loaded~~
- fixed GenerateCommand to not use old config options
  config options as 'oneFile' were still inside GenerateCommand and caused an error because 'oneFile' was removed in v3
- ~~added detection of autoload file inspired by phpunit
  i have wsdl2phpgenerator-cli as a require-dev dep in a project, if you would call: `php vendor/bin/wsdl2php ....` it would fail because the autoload wasnt in the assumed path.~~
 removed since not necessary anymore
- removed some config options that has been removed in v3

ivoba committed:
updated wsdl2phpgenerator to 3.1.2
fixed GenerateCommand to not use old config options
added detection of autoload file inspired by phpunit

ivoba committed:
removed removed option for v3

kasperg commented:
Thanks for taking a stab at getting the cli version of wsdl2phpgenerator 3 up and running again @ivoba. Apologies for the late reply.

The changes appear to a good starting point but [the current test cases are failing](https://travis-ci.org/wsdl2phpgenerator/wsdl2phpgenerator-cli/builds/48650449). I would appreciate if you could fix these and then I will take a closer look at the proposed changes.

ivoba committed:
fixed tests by removeing all tests that tested removed features of wsdl2phpgenerator

ivoba commented:
fixed tests, just removed tests for some removed features.
let me know if you need something else.

darxriggs commented:
Hi. Just wanted to ask what hinders this PR to be merged?
I am still using wsdl2phpgenerator v2.x due to the not yet updated command line application.
Looking forward to it.

speto commented:
@kasperg Any chance to move on with this?

ivoba committed:
Merge remote-tracking branch 'upstream/3.x' into 3.x

ivoba committed:
- raised PHP to 5.5.3 since its the min PHP version of symfony 2.3
- updated symfony console to 2.6 since its the highest version that supports PHP 5.3.3
- added PHP7 & PHP7.1 to travis
- some cleanups regarding wsdl2phpgenerator v3
- updated dependencies where possible

ivoba committed:
symfony/console version fix

ivoba commented:
@kasperg i updated the PR to keep track with lastest wsdl2phpgenerator version.
see updated PR description.

if you need help with maintenance of the cli application, let me know.

ivoba committed:
added soapClientClass option

kasperg commented:
@ivoba Apologies for the late reply. Thanks for the suggestions and the updates over time. The latest version addresses most of the updates made in v3. 

I appreciate help maintaining the CLI application. I have sent you an invite to join the team.
